lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -54,6 +54,18 @@ class GoogleCloudOutput < BufferedOutput
     config_param :zone, :string, :default => nil
     config_param :vm_id, :string, :default => nil
 
+    # List of parsed fields whose values are sent as labels instead of
+    # becoming part of the struct payload.
+    #
+    # This is specified as a JSON array where the 'name' is the field name
+    # parsed by fluentd and the 'value' is the name of the label to send.
```

Comment:
The JSON documentation refers to these as name/value pairs...  but it is confusing, I re-wrote this comment and I think it's more understandable, but we'll have to have Sam write some documentation with more examples.